packages/core-flows/src/shipping-profile/steps/delete-shipping-profile.ts

```diff
@@ -0,0 +1,28 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IFulfillmentModuleService } from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+export const deleteShippingProfilesStepId = "delete-shipping-profile"
+export const deleteShippingProfilesStep = createStep(
+  deleteShippingProfilesStepId,
+  async (ids: string[], { container }) => {
+    const service = container.resolve<IFulfillmentModuleService>(
+      ModuleRegistrationName.FULFILLMENT
+    )
+
+    await service.softDeleteShippingProfiles(ids)
+
+    return new StepResponse(void 0, ids)
```

Comment:
todo: Return IDs so we can remove links.

Reply:
We have these ids as the workflow input. Does it seem unnecessary to store the result of this step since we already have the data in the workflow? Also, we use this "pattern" in all delete steps so far.

Reply:
Got it, let's keep it as is